packages/app/src/app/overmind/namespaces/editor/internalActions.ts

```diff
@@ -40,7 +40,7 @@ export const initializeSandbox = async (
   sandbox: Sandbox
 ) => {
   await Promise.all([
-    actions.editor.internal.initializeLiveSandbox(sandbox),
+    // actions.editor.internal.initializeLiveSandbox(sandbox),
```

Comment:
@necoline I have removed the initialization of live sessions on load so the users see the "Go Live" by default, should I keep it?

Reply:
Hmm, I think we want them to see the button to go live for now since more visibility on the feature that will me moved behind a paywall will encourage people to think about whether on not, they should think about upgrading